docs/content/direct/get-started.md

```diff
@@ -14,6 +14,10 @@ This page shows one concrete example of steps 2--7 from the [full Installation a
 
 This is one way to produce a very simple system, suitable for study but not production usage. For general setup information, see [the full story](user-guide-intro.md#the-full-story).
 
+### Note for Windows users
+
+Someone using WSL [had a problem with the recipe on this page and was able to work around it by running the whole recipe as the `root` user in Linux](knownissue-wsl-ghcr-helm.md).
```

Comment:
'someone'? Why are we pointing out a person in our documentation? 

"For some users on WSL, use of our getting started guide and demo environment creation script may require running with the `root` user."

Reply:
Reworded.